components/Feedback.js

```diff
@@ -0,0 +1,110 @@
+                    </div>
+                </a>
+            </div>
+        )
+    }
+    if(error){
+        return (
+            <div className={`flex flex-col rounded-md shadow-md text-center border border-gray-200 p-4 ${className}`}>
+            <div className='block mx-auto w-fit'>
+                <img src='/img/illustrations/icons/icon-x.svg' className='md:w-14' />
+            </div>
+            <div className='text-center mx-auto text-lg mt-4'>
+                Oops! Something went wrong...
+            </div>
+            <div className='text-center mx-auto text-md text-gray-500'>
+                We were unable to process your feedback
+            </div>
+            <a className='bg-black text-white flex flex-row justify-center text-center shadow-md hover:shadow-lg transition-all duration-500 ease-in-out py-2 px-4 rounded mx-auto mt-4' href='https://github.com/asyncapi/website/issues/new?assignees=alequetzalli+-&labels=%F0%9F%93%91+docs&template=docs.yml&title=%5B%F0%9F%93%91+Docs%5D%3A+' target='_blank' rel='noopener noreferrer'>
+                <img src='/img/logos/github-fill.svg' className='w-6 mr-2' />
+                Create Issue on Github
+            </a>
+        </div>
+        );
+    }
+    return (
+        <div className={`flex flex-col rounded-md shadow-md border border-gray-200 p-4 ${className}`}>
+            <div className='flex flex-row'>
+                <img src="/img/illustrations/icons/icon.svg" className='w-28 md:w-14' />
+                <div className='flex flex-col ml-4'>
+                    <div className='text-xl'>
+                        Was this Helpful?
+                    </div>
+                    <div className='text-sm text-gray-500'>
+                        Help us improve the docs by adding your contribution.
+                    </div>
+                </div>
+            </div>
+            <form onSubmit={handleSubmit}>
+                <div className='flex flex-col my-4'>
+                    <textarea className='w-full h-20 border rounded-md bg-gray-50 text-sm px-2 py-1 text-gray-700 inline-block align-top focus:border-0' placeholder='Write your suggestions here' onChange={(e) => setFeedback(e.target.value)} required />
+                    <div className='block lg:flex lg:flex-row mt-4 text-sm'>
+                        <button className='bg-primary-500 text-white w-full lg:w-6/12 py-2 shadow-md hover:shadow-lg transition-all duration-500 ease-in-out rounded' type='submit'>Submit feedback</button>
+                        <div className='my-2 text-center font-medium w-full lg:w-1/12 lg:my-auto'>OR</div>
+                        <a className='bg-black text-white flex flex-row lg:w-6/12 shadow-md hover:shadow-lg transition-all duration-500 ease-in-out py-2 rounded justify-center' href='https://github.com/asyncapi/website/issues/new?assignees=alequetzalli+-&labels=%F0%9F%93%91+docs&template=docs.yml&title=%5B%F0%9F%93%91+Docs%5D%3A+' target='_blank' rel='noopener noreferrer'>
```

Comment:
we duplicate this as I can see, can we have it as single component?

Reply:
Do you want a separate component for buttons that are displayed in this feedback box?

Reply:
yes, please

Reply:
@derberg I have updated the PR accordingly. Kindly review the PR and do ping if certain changes are needed more.